sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -274,6 +280,17 @@ private void printSystemInformation(StringBuilder stringBuilder) {
         }
     }
 
+    private void printTransportRequestTimeline(StringBuilder stringBuilder) {
+        if (transportRequestTimeline != null) {
+            stringBuilder.append("Transport request timeline -------").append(System.lineSeparator());
+            Iterator<RequestTimeline.Event> iterator = transportRequestTimeline.iterator();
+            while (iterator.hasNext()) {
+                RequestTimeline.Event event = iterator.next();
+                stringBuilder.append("  eventName = " + event.getName() + ",  startTime = " + event.getStartTime() + ",  durationInMicrosec = " + event.getDuration().toNanos()/1000L).append(System.lineSeparator());
```

Comment:
do we have a specific format for request timeline? is it going to be json?

Reply:
After the latest merge , it will be json , sample is in the PR description for both rntbd and http